agent/stats/queue_test.go

```diff
@@ -785,3 +786,97 @@ func TestPerSecNetworkStatSetFailWithOneDatapoint(t *testing.T) {
 	stats, err := queue.GetNetworkStatsSet()
 	require.Errorf(t, err, "Received unexpected network stats set %v", stats)
 }
+
```

Comment:
Could we add a unit test in here to directly test the new `AddContainerStat` function?

Reply:
I will add this in a follow up pull request. The existing `Add` function (which `AddContainerStat` calls) does not have a unit test that directly calls it either, so I will add a unit test for that as well in the follow up. 👍

Reply:
Follow up pull request: https://github.com/aws/amazon-ecs-agent/pull/4208